README.md

```diff
@@ -20,3 +20,35 @@ To specify the table title for a machine, use the class feature of cukinia : `lo
 
 The source directory can be selected with the `-i` option. This directory shall contain all xml file to include to the test report.
 It can also include a `prerequisites.adoc` file that will be added on top of the document and a `notes.adoc` file that will appear at the end. Both files are optional.
+
+## Compliance matrix
+
+This feature only works with the id feature provided by the `-s` option.
+
+One or many compliance matrices can be added at the end of the report. These matrices are auto-generated by the program, using a csv file in entry.
+
+The csv must contain on each line a requirement and a test id. If a requirement concerns many test ids, it must appear on many lines.
+Here is an example :
+
+```
+cluster requirement,289
+security requirement,143
+security requirement,17
+security requirement,23
+stress resistance,34
+stress resistance,35
+documentation requirement,345
+```
+
+Please note that for now, all lines concerning the same requirement shall be put one after another. The format should be respected to ensure a correct output.
```

Comment:
Isn't this fixed by sorting?
If not, fix it and remove this part of the documentation.

Reply:
Yes it is, this README is outdated and was changed in the last commit.